### PR TITLE
feat: Reorder Carrere Stage tags

### PR DESCRIPTION
**Pull request by MartinBelton-gov**
Reorder Carrere Stage tags

### CODE REVIEW

File: browser-tests/content-regression-tests/tests/resources-learning/search.spec.ts [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import { test, expect } from '@playwright/test'
 
-test.describe.skip('Resources and learning', () => {
+test.describe('Resources and learning', () => {
```
Comment (killij) [inline browser-tests/content-regression-tests/tests/resources-learning/search.spec.ts:3]: Need to add the skip to the whole set of tests, it's only been added to a single test here

**Review by killij — CHANGES_REQUESTED** [review]
Just need to sort the test exclusions

File: browser-tests/content-regression-tests/tests/resources-learning/search.spec.ts (hunk 1 above)
Comment (MartinBelton-gov) [reply browser-tests/content-regression-tests/tests/resources-learning/search.spec.ts]: Corrected the skip (now on the original test)

**Review by killij — APPROVED** [review]
G2G

**Pull request merged**